cosmo_tester/test_suites/snapshots/__init__.py

```diff
@@ -91,11 +91,10 @@ def get_multi_tenant_versions_list():
         return MULTI_TENANT_MANAGERS
 
 
-def upgrade_agents(cfy, manager, logger, tenants=('default_tenant',)):
+def upgrade_agents(cfy, manager, logger):
     manager.use()
-    for tenant in tenants:
-        _log('Upgrading agents', logger, tenant)
-        cfy.agents.install(['-t', tenant])
+    logger.info('Upgrading agents')
+    cfy.agents.install(['--all-tenants'])
```

Comment:
is this actually ok? @geokala

Reply:
It wasn't on community tests- can you try running 'cfy agents install --all-tenants' on the latest community manager and see if it complains?

Reply:
Trying now